lib/xlsx/xform/sheet/data-validations-xform.js

```diff
@@ -51,7 +51,15 @@ function optimiseDataValidations(model) {
   return dvList
     .map(dv => {
       if (!dv.marked) {
-        const addr = colCache.decodeAddress(dv.address);
+        const addr = colCache.decodeEx(dv.address);
+        if (addr.dimensions) {
+          // If we comeback with a dimensions, we can assume this was sone on purpose and should not be optimized
```

Comment:
I think it's fine to remove this comment

Reply:
Done